src/manifest.json

```diff
@@ -15,8 +15,27 @@
   },
   "content_scripts": [
     {
-      "matches": ["https://*/search*"],
-      "include_globs": ["*.google.*/*"],
+      "matches": [
+        "https://*/search*",
+        "https://duckduckgo.com/*q=*",
+        "https://*.startpage.com/sp/search*",
+        "https://*.baidu.com/s*wd=*"
```

Comment:
I think we can change this to `https://*/*`?

Reply:
yes, we can